test/from-markdown/custom-blocks/empty-tag/input.md

```diff
@@ -1,8 +1,12 @@
-Hello world
+A
 
 {% %}
 
+B
+
```

Comment:
I just updated this test to make sure invalid tags are simply ignored, but the content between is kept.